appveyor.yml

```diff
@@ -1,6 +1,6 @@
 version: 1.0.{build}
 skip_branch_with_pr: true
-image: Visual Studio 2017
```

Comment:
Nice, I didn't know you could do that....

Reply:
See https://www.appveyor.com/updates/.

There should be a way to set the specific update to use. Otherwise, it may break again after the next update.